tests/python/gaia-ui-tests/gaiatest/tests/functional/music_oga/manifest.ini

```diff
@@ -0,0 +1,26 @@
+[parent:../manifest.ini]
```

Comment:
I wonder if we really need to keep the tests of Music OGA. If we had to roll back to the OLD OGA, A backout of this commit would suffice. What do you think?